OpenRA.Game/Server/Server.cs

```diff
@@ -323,9 +364,9 @@ internal void OnConnectionPacket(Connection conn, int frame, byte[] data)
 			events.Add(new ConnectionPacketEvent(conn, frame, data));
 		}
 
-		internal void OnConnectionPing(Connection conn, int[] pingHistory, byte queueLength)
```

Comment:
I know that we've hit a roadblock on the new netcode, but its a bit early to start ripping out its features before we are even able to use them... 😉